tap_bitbucket/gitlocal.py

```diff
@@ -178,8 +201,19 @@ def _cloneRepo(self, repo, repoWdir):
     """
     Clones a repository using git clone, throwing an error if the operation does not succeed
     """
+    cloneUrl = self.sourceUrlPattern.format(self.token, repo)
+    
     # If directory already exists, do an update
     if os.path.exists(repoWdir):
+      logger.info("Updating git remote")
```

Comment:
We don't currently use this, but I noticed it while debugging.  When a repo is already checked out, the token can expire and the remote won't let you update. This updates the remote to ensure `git remote update` is successful.